docs/docs/architecture/orchestration.md

```diff
@@ -73,14 +70,14 @@ The *clusterID* uniquely identifies a cluster and can be used to [verify your cl
 
 ## Upgrades
 
-Constellation images and components might need to be upgraded to new versions during the lifetime of a cluster.
+Constellation images and components may need to be upgraded to new versions during the lifetime of a cluster.
 Constellation implements a rolling update mechanism ensuring no downtime of the control or data plane.
 You can upgrade a Constellation cluster with a single operation by using the CLI.
-For step-by-step instructions on how to do this, refer to [Upgrade Constellation](../workflows/upgrade.md).
+For step-by-step instructions on how to do this, refer to [Upgrade your cluster](../workflows/upgrade.md).
 
 ### Attestation of upgrades
 
-The new verification hashes (measurements) are released together with every image.
+With every new image, corresponding measurements are released.
 During an update procedure, the CLI provides the new measurements to the [JoinService](components.md#joinservice) securely.
 New measurements for an updated image are automatically pulled and verified by the CLI following the [supply chain security concept](attestation.md#chain-of-trust) of Constellation.
 The [attestation section](attestation.md#cluster-facing-attestation) describes in detail how these measurements are then used by the JoinService for the attestation of nodes.
```

Comment:
Can you explain what the last sentence on this page means? (can't comment on that line)

Reply:
```markdown
### Attestation of upgrades

With every new image, corresponding measurements are released.
During an update procedure, the CLI provides the new measurements to the [JoinService](components.md#joinservice) securely.
New measurements for an updated image are automatically pulled and verified by the CLI following the [supply chain security concept](attestation.md#chain-of-trust) of Constellation.
The [attestation section](attestation.md#cluster-facing-attestation) describes in detail how these measurements are then used by the JoinService for the attestation of nodes.

The updated measurements are reproducible based on the updated node images, hence,  auditable by the customer.
```
@m1ghtym0 I don't get the last sentence. Can you explain?

Reply:
Should say: A user can verify the new image's measurements by reproducing them locally using our reproducible image builds

Reply:
Is my rephrasing ok?